src/components/dialogs/payment-complete.vue

```diff
@@ -109,7 +111,20 @@ export default class PaymentCompleteDialog extends Mixins(
   }
 
   async fetchNr (): Promise<void> {
-    const nrData = await NamexServices.getNameRequest(true)
+    let nrData = await NamexServices.getNameRequest(true)
+    if (nrData.state === NrState.PENDING_PAYMENT && this.incompletePaymentRetries > 0) {
```

Comment:
I haven't tested this on a true `PENDING_PAYMENT` will try it now, perhaps there are additional conditions we should check here?

Reply:
🤷‍♂️

Reply:
![image](https://github.com/bcgov/namerequest/assets/3484109/77b893c3-040a-471b-8e2b-4179d3bec518)

Reply:
I'll probably let this one sit overnight and think about it